MANIFEST.in

```diff
@@ -0,0 +1,4 @@
+include LICENSE
```

Comment:
A quoi sert ce fichier ?

Reply:
Quand tu crée un package, [un nombre limité de fichier est inclus dans l'archive.](https://packaging.python.org/guides/using-manifest-in/). Le fichier MANIFEST.in permet d'ajouter des fichiers qui sont absents par défaut, ou d'exclure des fichiers présents par défaut. L'ensemble des fichiers présents dans ton package est répertorié dans `MAPIE.egg-info/SOURCES.txt`, après que tu aies exécuté la command `python setup.py develop` ou `python setup.py sdist`